### ISSUE TITLE
Error for dependency tableexport.jquery.plugin

### ISSUE THREAD

**Issue by itatdcer** [webjars/webjars#1962]
Trying to use 
`<dependency>
    <groupId>org.webjars.npm</groupId>
    <artifactId>tableexport.jquery.plugin</artifactId>
    <version>1.21.0</version>
</dependency>`
leaves maven hanging up.

**Comment by itatdcer** [comment]
No, beside some WARNIGS regarding missing dependencies (could be related?).
maven log:
`$ mvn clean package
[INFO] Scanning for projects...
[INFO] 
[INFO] -------------------------< XXX:XXXXXX >-------------------------
[INFO] Building XXXXXX 1.0.0-SNAPSHOT
[INFO] --------------------------------[ jar ]---------------------------------
[WARNING] The POM for org.webjars.npm:hosted-git-info:jar:2.6.0 is missing, no dependency information available
[WARNING] The POM for org.webjars.npm:semver:jar:2- >=2.2.1,[3,4),[4,5),[5,6) is missing, no dependency information available
[WARNING] The POM for org.webjars.npm:gauge:jar:2.7.4 is missing, no dependency information available
[WARNING] The POM for org.webjars.npm:cyclist:jar:0.2.2 is missing, no dependency information available
[WARNING] The POM for org.webjars.npm:mime:jar:1.2.2 is missing, no dependency information available
[WARNING] The POM for org.webjars.npm:es6-set:jar:0.1.5 is missing, no dependency information available`

After that, maven hangs forever and does nothing.

**Comment by jamesward** [comment]
I'm redeploying this on https://webjars.org to see if there are some missing dependencies that need to be deployed.  In a few hours maybe you can try again.  Also, you could try version `1.22.0` to see if that helps.

**Comment by itatdcer** [comment]
Thanks for your help.

I've tried bot versions with  the same outcome.
